src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DueDiligenceDataBlockValidatorTest.java

```diff
@@ -1,110 +1,122 @@
 package uk.gov.companieshouse.overseasentitiesapi.validation.utils;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.companieshouse.overseasentitiesapi.mocks.DueDiligenceMock;
 import uk.gov.companieshouse.overseasentitiesapi.mocks.OverseasEntityDueDiligenceMock;
 import uk.gov.companieshouse.overseasentitiesapi.model.dto.DueDiligenceDto;
 import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntityDueDiligenceDto;
+import uk.gov.companieshouse.overseasentitiesapi.validation.DueDiligenceValidator;
+import uk.gov.companieshouse.overseasentitiesapi.validation.OverseasEntityDueDiligenceValidator;
 import uk.gov.companieshouse.service.rest.err.Err;
 import uk.gov.companieshouse.service.rest.err.Errors;
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.DueDiligenceDataBlockValidator.QUALIFIED_FIELD_NAMES;
 
+@ExtendWith(MockitoExtension.class)
 class DueDiligenceDataBlockValidatorTest {
 
    private static final String LOGGING_CONTEXT = "12345";
+   @InjectMocks
+   private DueDiligenceDataBlockValidator dueDiligenceDataBlockValidator;
+   @Mock
+   private OverseasEntityDueDiligenceValidator overseasEntityDueDiligenceValidator;
+   @Mock
+   private DueDiligenceValidator dueDiligenceValidator;
 
-   private DueDiligenceDataBlockValidator dueDiligenceDataBlockValidator = new DueDiligenceDataBlockValidator();
    private OverseasEntityDueDiligenceDto overseasEntityDueDiligenceDto = OverseasEntityDueDiligenceMock.getOverseasEntityDueDiligenceDto();
    private DueDiligenceDto dueDiligenceDto = DueDiligenceMock.getDueDiligenceDto();
 
```

Comment:
I'd also add 2 tests to verify that dueDiligenceDataBlockValidator or overseasEntityDueDiligenceValidator gets called when either one of those blocks is present.

Reply:
Added the verification tests